xml/abstract_commands.xml

```diff
@@ -94,4 +94,82 @@
         <field name="0" bits="23:0" />
     </register>
 
+    <register name="Access Memory">
+        \begin{steps}{This command gives the debugger access to memory
+            locations, including memory-mapped registers. It performs the
+            following sequence of operations:}
+        \item Copy data from the memory location specified in {\tt arg1} into the
+            {\tt arg0} portion of {\tt data}, if \Fwrite is clear and \Ftransfer is set.
+        \item Copy data from the {\tt arg0} portion of {\tt data} into the
+            memory location specified in {\tt arg1}, if \Fwrite is set and \Ftransfer is set.
+        \item Increment {\tt arg1}, if \Fpostincrement is set.
+        \item Execute the Program Buffer, if \Fpostexec is set.
+        \end{steps}
+
+        If any of these operations fail, \Fcmderr is set and none of the
+        remaining steps are executed. An implementation may detect an upcoming
+        failure early, and fail the overall command before it reaches the step
+        that would cause failure.
+
+        Debug Modules may optionally implement this command
+        and may support read and write access to memory locations when the selected hart is running or halted.
+        If this command supports memory accesses
+        while the hart is running, it must also
+        support memory accesses while the hart is halted.
+
+        \begin{commentary}
+            The encoding of \Fsize was chosen to match \Fsbaccess in \Rsbcs.
+        \end{commentary}
+
+        <field name="cmdtype" bits="31:24">
+            This is 2 to indicate Access Memory Command.
+        </field>
+        <field name="0" bits="23" />
+        <field name="size" bits="22:20">
+            0: Access the lowest 8 bits of the memory location.
```

Comment:
what about misalignment? Is it an error? Undefined?

Reply:
I'm adding a sentence that basically says that the behavior is whatever it is if the hart is running M mode code.